modules/standard/UtilReplicatedVar.chpl

```diff
@@ -115,10 +144,18 @@ proc rcCollect(replicatedVar: [?D] ?MYTYPE, collected: [?CD] MYTYPE): void
       col = replicatedVar[rcDomainIx];
 }
 
+/*
+Access the copy of `replicatedVar` on the current locale.
+
+This is equivalent to ``replicatedVar[1]``.
```

Comment:
Doesn't ``replicatedVar[1]`` give you the element at index 1?  Can you explain the utility of this to me further?  I'm having difficulty understanding it at the moment.